gulpfile.js

```diff
@@ -74,6 +74,28 @@ gulp.task('umd', function(cb) {
       webpackCallBack('webpack', cb));
 });
 
+gulp.task('npm', function() {
+  var pkgJson = require('./package.json');
+  var targetPkgJson = {};
+  var fieldsToCopy = ['version', 'description', 'keywords', 'author', 'repository', 'license', 'bugs', 'homepage'];
+
+  targetPkgJson['name'] = '@ng-bootstrap/ng-bootstrap';
```

Comment:
Are we sure this work? I mean, I have no idea in how to tell npm that we want a scoped package like that.